src/Momento.Sdk/Internal/LoggingUtils.cs

```diff
@@ -379,8 +379,6 @@ public static TSuccess LogTraceCollectionRequestSuccess<TSuccess>(this ILogger _
             }
             return success;
         }
-        
-#if NETSTANDARD2_0_OR_GREATER
```

Comment:
None of these NETSTANDARD2.0 directives are necessary anymore since I bumped the target framework from .NET 4.61 to .NET 4.62.